Content.Shared/Implants/SharedImplanterSystem.cs

```diff
@@ -226,3 +284,39 @@ public AddImplantAttemptEvent(EntityUid user, EntityUid target, EntityUid implan
         Implanter = implanter;
     }
 }
+
+
+[Serializable, NetSerializable]
+public sealed class DeimplantBuiState : BoundUserInterfaceState
+{
+    public readonly string? Implant;
+
+    public Dictionary<string, string> ImplantList;
+
+    public DeimplantBuiState(string? implant, Dictionary<string, string> implantList)
+    {
+        Implant = implant;
+        ImplantList = implantList;
+    }
+}
+
```

Comment:
Better to use compstates for this stuff if possible. DeimplantChosen is networked anyway.

Reply:
I'm not sure how I would use component states to send UI updates. I couldn't find any existing implementation that does this to use as a base, so if you got any examples I'd appreciate it.